plugins/node/opentelemetry-plugin-ioredis/test/ioredis.test.ts

```diff
@@ -48,8 +48,8 @@ const DEFAULT_ATTRIBUTES = {
   [GeneralAttribute.NET_PEER_ADDRESS]: URL,
 };
 
-const okStatus: Status = {
-  code: CanonicalCode.OK,
+const spanStatus: Status = {
```

Comment:
In all other places, it is called `unsetStatus`.
It can be nice to keep consistency :)